tough/src/key_source.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
It'd be nice if we could remove these across tough as well, similar to Bottlerocket.

Reply:
Sorry, I should have been clearer - I don't have authority to request this, but if we can get it approved, this would be nice to have (in a separate commit).

Reply:
Looks like I successfully 'untouched' all of the copyright headers.